testing/game/Game003 - take hint.md

```diff
@@ -26,7 +26,7 @@
 |Click on Ziskat napovedu (-5 bodu).|The confirmation dialog is displayed.|
 |Decline the confirmation dialog.|The hint page is displayed.<br>No hint is taken.|
 |Click on Ziskat napovedu (-5 bodu).|The confirmation dialog is displayed.|
-|Confirm the dialog.|The dialog is closed.<br>The hintA is taken.<br>The cypher detail is displayed.|
+|Confirm the dialog.|The dialog is closed.<br>The hintA is taken.<br>~~The cypher detail is displayed~~.<br>Go to cypher detail page|
```

Comment:
v results sloupecku by se nemela delat zadna akce, pouze overovani, takze to "Go to cypher detail page" bych dal spise do nasledujiciho kroku:

`|Go to cypher detail page.<br>Verify that the hintA text is displayed under the map.|The hintA text is present.|`